src/directLine.ts

```diff
@@ -438,7 +438,9 @@ export class DirectLine implements IBotConnection {
         const parsedPollingInterval = ~~options.pollingInterval;
 
         if (parsedPollingInterval < POLLING_INTERVAL_LOWER_BOUND) {
-            console.warn(`DirectLineJS: provided pollingInterval (${ options.pollingInterval }) is under lower bound (200ms), using default of 1000ms`);
+            if (typeof options.pollingInterval !== 'undefined') {
```

Comment:
Why typeof, and not just `options.pollingInterval === undefined`?

Reply:
This is a JS idiom used because in some cases the identifier being check may not have been initialized yet. That's not the case here, but it doesn't hurt using `typeof`.